### ISSUE TITLE
I'm getting the following stack trace

### ISSUE THREAD

**Issue by kicker69101** [istrategylabs/django-rq-scheduler#12]
I'm getting the following stack trace when I try to add anything to the schedule or repeatable tasks:

```
Environment:


Request Method: GET
Request URL: http://localhost:8080/admin/scheduler/repeatablejob/add/

Django Version: 1.9.7
Python Version: 2.7.8
Installed Applications:
('django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'debug_toolbar',
 'rest_framework',
 'frontend',
 'backups',
 'nas',
 'django_rq',
 'scheduler')
Installed Middleware:
['debug_toolbar.middleware.DebugToolbarMiddleware',
 'django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware']



Traceback:

File "/Users/someuser/.pyenv/versions/refreshesv3/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  149.                     response = self.process_exception_by_middleware(e, request)

File "/Users/someuser/.pyenv/versions/refreshesv3/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  147.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/Users/someuser/.pyenv/versions/refreshesv3/lib/python2.7/site-packages/django/contrib/admin/options.py" in wrapper
  541.                 return self.admin_site.admin_view(view)(*args, **kwargs)

File "/Users/someuser/.pyenv/versions/refreshesv3/lib/python2.7/site-packages/django/utils/decorators.py" in _wrapped_view
  149.                     response = view_func(request, *args, **kwargs)

File "/Users/someuser/.pyenv/versions/refreshesv3/lib/python2.7/site-packages/django/views/decorators/cache.py" in _wrapped_view_func
  57.         response = view_func(request, *args, **kwargs)

File "/Users/someuser/.pyenv/versions/refreshesv3/lib/python2.7/site-packages/django/contrib/admin/sites.py" in inner
  244.             return view(request, *args, **kwargs)

File "/Users/someuser/.pyenv/versions/refreshesv3/lib/python2.7/site-packages/django/contrib/admin/options.py" in add_view
  1437.         return self.changeform_view(request, None, form_url, extra_context)

File "/Users/someuser/.pyenv/versions/refreshesv3/lib/python2.7/site-packages/django/utils/decorators.py" in _wrapper
  67.             return bound_func(*args, **kwargs)

File "/Users/someuser/.pyenv/versions/refreshesv3/lib/python2.7/site-packages/django/utils/decorators.py" in _wrapped_view
  149.                     response = view_func(request, *args, **kwargs)

File "/Users/someuser/.pyenv/versions/refreshesv3/lib/python2.7/site-packages/django/utils/decorators.py" in bound_func
  63.                 return func.__get__(self, type(self))(*args2, **kwargs2)

File "/Users/someuser/.pyenv/versions/refreshesv3/lib/python2.7/site-packages/django/utils/decorators.py" in inner
  184.                     return func(*args, **kwargs)

File "/Users/someuser/.pyenv/versions/refreshesv3/lib/python2.7/site-packages/django/contrib/admin/options.py" in changeform_view
  1367.         ModelForm = self.get_form(request, obj)

File "/Users/someuser/.pyenv/versions/refreshesv3/lib/python2.7/site-packages/scheduler/admin.py" in get_form
  16.         queue_field = obj._meta.get_field('queue')

Exception Type: AttributeError at /admin/scheduler/repeatablejob/add/
Exception Value: 'NoneType' object has no attribute '_meta'
```

I'm not sure what I did wrong here and I'm hoping I could get some help.

**Comment by chassing** [comment]
@kicker69101 see my PR (https://github.com/istrategylabs/django-rq-scheduler/pull/10). It's a bug

**Comment by g3rd** [comment]
Fixed with #10 Thanks @chassing
